VAMobile/src/screens/HealthScreen/Appointments/AppointmentTypeComponents/SharedComponents/AppointmentLocation.tsx

```diff
@@ -43,12 +44,26 @@ const locationHeading = (subType: AppointmentDetailsSubType, type: AppointmentDe
         case AppointmentDetailsTypeConstants.VideoVA:
         case AppointmentDetailsTypeConstants.VideoAtlas:
           return t('appointments.location.title')
+        case AppointmentDetailsTypeConstants.CommunityCare:
+          return t('providerInformation')
         default:
-          undefined
+          return undefined
       }
   }
 }
 
+const getCommunityCareProvider = (attributes: AppointmentAttributes, type: AppointmentDetailsScreenType) => {
+  const { healthcareProvider } = attributes
+
+  if (type === AppointmentDetailsTypeConstants.CommunityCare && healthcareProvider) {
+    return (
+      <TextView variant="MobileBody" selectable={true}>
+        {healthcareProvider}
```

Comment:
For now I'm showing healthcareProvider here. Based on @rbontrager's findings we may need to request that the BE team add a different field to show instead, but we aren't sure yet